src/main.rs

```diff
@@ -275,6 +320,43 @@ async fn setup_eth_bridge<'a>(
     eth_bridge_setup_outputs
 }
 
+async fn upgrade_eth_bridge<'a>(
+    account: Option<RpcAccount<'a>>,
+    config_file: &ConfigFile,
+    clients: &Clients,
+) -> color_eyre::Result<()> {
+    let account = match account {
+        Some(account) => account,
+        None => get_account(clients, config_file).await,
+    };
+    upgrade_eth_token_to_cairo_1(
+        &account,
+        clients.provider_l2(),
+        Felt::from_str(
+            &config_file.l2_eth_token_proxy_address.clone().expect("l2_eth_token_proxy_address not in config."),
```

Comment:
do we want expects or should we throw errors? i think we had moved a lot of expects in the main file so i am in the camp of throwing errors and keeping it coherent

Reply:
let's create a separate issue for this ? or should I implement this in this PR itself ?

Reply:
i think this we should do in this PR